### ISSUE TITLE
Please provide an Android version

### ISSUE THREAD

**Issue by mquinson** [seanh/storymaps#3]
Would it make any sense to have a simplified version for Android? Maybe something like a little game where the computer picks a bunch of cards, and you have to come up with a story along these lines. This would make a great game to play with the kids, for example in waiting queues.

Thanks in advance,

**Comment by seanh** [comment]
I think a simplified version for Android is a great idea!

Are you thinking of phones or tablets?

After using StoryMaps in schools I realised that it would work fantastically on tablets (which didn't exist when I first wrote it) if the text editor was replaced with an audio recorder. The kids love looking at the story cards, zooming in and out, deciding which ones to have, dragging and dropping them into the order that they want - all stuff that will work great on a tablet.

But when it comes to the actual write your story by typing it in part - typing is very difficult and time consuming for children. I usually had them print out their stories and write on the paper, but writing is also difficult and time consuming.

So I'd love to replace the whole text editor portion of the UI with a simple audio recorder: the only buttons it would need, on each story card, would be record and play back (with the ability to re-record over the original recording if you messed up). It wouldn't need actual audio editing. The idea is that the kids record one short audio clip for each story card they've chosen. Then the app could play back their story by showing the cards full screen one by one while playing the audio clip for each card.

I like your idea of a mode where the computer picks the cards for you as well. That would work especially well on phones where the whole choosing and ordering your cards UI wouldn't fit on the screen. But it'd be a great mode on tablets and desktops as well.

**Comment by mquinson** [comment]
Yes, I was thinking of phones. But actually, my thoughts were still very preliminary... I never used any of this with kids, because of the language barrier. I was just dreaming of having some light version usable in any settings, just with a simple phone. I'm sure you have much better ideas on how to put my dime dream into practice... 
Thanks,
